pyroomacoustics/datasets/google_speech_commands.py

```diff
@@ -0,0 +1,209 @@
+# -*- coding: utf-8 -*-
+'''
+Google's Speech Commands Dataset
+======================
+The Speech Commands Dataset has 65'000 one-second long utterances of 30 short 
+words, by thousands of different people, contributed by members of the public
+through the AIY website. It’s released under a Creative Commons BY 4.0 license.
+
+More info about the dataset can be found at the link below:
+
+https://research.googleblog.com/2017/08/launching-speech-commands-dataset.html
+
+AIY website for contributing recordings:
+
+https://aiyprojects.withgoogle.com/open_speech_recording
+
+Tutorial on creating a word classifier:
+
+https://www.tensorflow.org/versions/master/tutorials/audio_recognition
+'''
+
+import os, glob
+import numpy as np
+from scipy.io import wavfile
+
+try:
+    import sounddevice as sd
+    have_sounddevice = True
+except:
+    have_sounddevice = False
+
+from .utils import download_uncompress
+from .base import Meta, AudioSample, Dataset
+
+
+google_speech_commands_sounds = {
```

Comment:
I guess here `True/False` might be more explicit than `1/0`.

Reply:
`1` or I mean `True` ;)

I'll change to this.

Reply:
Ha ha :)